cfgov/teachers_digital_platform/documents.py

```diff
@@ -32,6 +38,7 @@ class ActivityPageDocument(Document):
     objectives = fields.TextField()
     what_students_will_do = fields.TextField()
     related_text = fields.TextField()
+    file_titles = fields.KewordField()
```

Comment:
@mrclay Is this a typo?

Reply:
Yep, and I guess I want a TextField because I'm concatting them to one string.